### PR TITLE
update patching doc

### PR DESCRIPTION
**Pull request by rosemarymarano**
As per Monica, OWLS-89082 Add new WKO documentation which describe how to patch WebLogic images using WIT

### CODE REVIEW

File: documentation/staging/content/userguide/base-images/_index.md [hunk 1]
```diff
@@ -154,7 +155,21 @@ including keys and credentials that are used to access external resources
...
 
+Use the [WebLogic Image Tool](https://github.com/oracle/weblogic-image-tool) (WIT) to patch
+WebLogic Server images with quarterly Patch Set Updates (PSUs), which include security fixes, or with one-off patches.
+
+Use either the WIT [`create`](https://github.com/oracle/weblogic-image-tool/blob/master/site/create-image.md) or
+[`update`](https://github.com/oracle/weblogic-image-tool/blob/master/site/update-image.md) command, however,
+patching using the `create` command results in a smaller WebLogic Server image size.
+
+Example: Create an image named `sample:wls` with the WebLogic Server 12.2.1.4.0 slim installer, JDK 8u291, latest Oracle 7-slim,
+and latest PSU and recommended CPU and SPU patches applied.
```
Comment (rjeberhard) [inline documentation/staging/content/userguide/base-images/_index.md:168]: This sample command presupposes that the customer has downloaded the installer and the patches and placed them in the image tool's cache. We could debate if we should document all of those steps here, but if we don't then we do need to document that downloading these artifacts in advance is required.

Comment (ddsharpe) [reply documentation/staging/content/userguide/base-images/_index.md:168]: +1, can we just add a "...(you will need to download the WebLogic Server 12.2.1.4.0 installer and JDK installer prior to running the create command)."

Comment (rjeberhard) [reply documentation/staging/content/userguide/base-images/_index.md:168]: Yes, let's do that for now.

Comment (rosemarymarano) [reply documentation/staging/content/userguide/base-images/_index.md:168]: Yes. For details, see the WIT Quick Start guide, https://github.com/oracle/weblogic-image-tool/blob/master/site/quickstart.md). Does that work?

**Pull request merged**